api/__tests__/app.test.js

```diff
@@ -73,6 +73,74 @@ describe("/api/reviews", () => {
   });
 });
 
+describe("/api/reviews?category", () => {
+  test("Accepts a category query", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews.length).toBe(1);
+        body.reviews.forEach((review) => {
+          expect(review.category).toBe("dexterity");
+        });
+      });
+  });
+  test("Returns a 404 not found error when given an invalid category query", () => {
+    return request(app)
```

Comment:
We need another 200 test here for when given a valid category query that has no related reviews (in which case we'd expect an empty array response and no error, there should be at least one category this is true for in the test data)